classes/classes/Items/Consumable.as

```diff
@@ -31,6 +31,8 @@ package classes.Items
 		protected function get flags():DefaultDict { return kGAMECLASS.flags; }
 		protected function get camp():Camp { return kGAMECLASS.camp; }
 		protected function doNext(eventNo:Function):void { kGAMECLASS.doNext(eventNo); }
+		
+		protected var mutationsHelper:MutationsHelper = new MutationsHelper();
```

Comment:
Thats not needed. Calling mutations.restoreLegs(...) and so on would do the trick without yet another instance.